.circleci/config.yml

```diff
@@ -497,12 +569,25 @@ workflows:
             branches:
               only: master
 
+      - docker_image_build_dev:
```

Comment:
What's the reason having two separate jobs: `docker_image_build_dev` and `docker_system_tests_dev` as the neither is reused?

Reply:
The reason is that I am not sure about Circle CI constraints on required jobs. I.e. I remember from Circle CI docs that required jobs need same branch/tag filter, hence I kept it explicit for being on the safe side.

Reply:
> I remember from Circle CI docs that required jobs need same branch/tag filter

Yes, it is.